test_team_2.py

```diff
@@ -15,6 +15,29 @@ def test_crypto(self):
 
     def test_nrp(self):
         """Test nrp functionality"""
+        # positive testcase
+        nationality = ['American', 'Canadian', 'Russian']
+        religion = ['Christianity', 'Islam', 'Hinduism']
+        political_group = ['Liberal', 'Conservative', 'Socialist Parties']
+        for n in nationality:
+            for r in religion:
+                for p in political_group:
+                    test_string = "My NRP is " + n + "," + r + ", and" + p
+                    expected = "My NRP is <NRP>"
+                    actual = anonymize_text(test_string, ["NRP"])
+                    self.assertEqual(expected, actual)
+
```

Comment:
You seem to be on the right track! I think what might be causing your test case to fail/error is the way you're implementing your anonymize_text function